edge/edge.entry.js

```diff
@@ -79,20 +79,6 @@ Element.prototype.replaceWith = function replaceWith(...nodes) {
 	}
 };
 
-// polyfill IntersectionObserverEntry.prototype.isIntersecting
-// Snippet from https://github.com/WICG/IntersectionObserver/pull/224
-if (!IntersectionObserverEntry.prototype.hasOwnProperty('isIntersecting')) {
-	Object.defineProperty(IntersectionObserverEntry.prototype,
-		'isIntersecting', {
-			get() {
-				return this.intersectionRatio > 0;
-			},
-		});
-}
-
-if (typeof requestIdleCallback === 'undefined') {
```

Comment:
`requestIdleCallback` is a separate polyfill and AFAICS not implemented in Edge yet.

Reply:
Thanks, i've put it back in.